spec/student_spec_rb

```diff
@@ -0,0 +1,19 @@
+require './student'
+
+describe Student do
+  context 'create a student' do
+    student = Student.new(35, 'Emre', parent_permission: true)
+
+    it 'show name' do
+      expect(student.name).to eq 'Emre'
+    end
+
+    it 'and age' do
+      expect(student.age).to eq 35
+    end
+
+    it 'has permission' do
+      expect(student.parent_permission).to eq true
+    end
+  end
+end
```

Comment:
- [ ] kindly fix the file name for this file it should be `student_spec.rb` & not `student_spec_rb` 🙏